src/android/NewRelicCordovaPlugin.java

```diff
@@ -134,8 +137,8 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
                                     errorStack.length() > 4095 ? errorStack.substring(0, 4094) : errorStack);
                         }
 
-                        NewRelic.recordBreadcrumb("Mobile JS Errors", crashEvents);
-                        NewRelic.recordCustomEvent("Mobile JS Errors", "", crashEvents);
+                        NewRelic.recordBreadcrumb("JS Errors", crashEvents);
+                        NewRelic.recordCustomEvent("JS Errors", "", crashEvents);
```

Comment:
Changed to "JS Errors" for consistency with React-Native and [iOS](https://github.com/ndesai-newrelic/newrelic-cordova-plugin/blob/7f744ef7a3effeb2452c71a5b00667f57000d454/src/ios/NewRelicCordovaPlugin.m#L92-L93)